packages/office-ui-fabric-react/src/components/Breadcrumb/Crumb.classNames.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  FontSizes,
+  HighContrastSelector,
+  IStyle,
+  ITheme,
+  ScreenWidthMaxMedium,
+  ScreenWidthMaxSmall,
+  mergeStyleSets
+} from '../../Styling';
+import { memoizeFunction } from '../../Utilities';
+
+const MediumMediaQuery = `@media screen and (max-width: ${ScreenWidthMaxMedium}px)`;
+const SmallMediaQuery = `@media screen and (max-width: ${ScreenWidthMaxSmall}px)`;
+
+export interface ICrumbClassNames {
+  root: string;
+  chevron: string;
+  crumbButton: string;
+  crumbLabel: string;
+  overflowIcon: string;
+  textContent: string;
+}
+export const getClassNames = memoizeFunction((
+  theme: ITheme,
+  isCurrentItem?: boolean
+): ICrumbClassNames => {
+  const { palette, fonts } = theme;
+
+  // Sizing values.
+  const chevronNormalSize = 12;
+  const chevronSmallSize = 8;
+  const height = 40;
+
+  // Colors used by the crumb.
+  const crumbHoverBackground = palette.neutralLighter;
+  const crumbHoverColor = palette.neutralPrimary;
+  const crumbActiveBackground = palette.neutralTertiaryAlt;
+  const crumbActiveColor = palette.neutralPrimary;
```

Comment:
This is effectively set on text, right? Should some of these come from the semantic slots instead of the palette?

Reply:
Probably although right now i'm just trying to translate the sass.